bin/install-production-tools.sh

```diff
@@ -0,0 +1,14 @@
+echo "Installing Eepromtool"
+cd /home/user/
+wget https://github.com/shadow-robot/sr-build-tools/raw/$(echo $toolset_branch | sed 's/#/%23/g')/bin/eepromtool
```

Comment:
toolset_branch should be passed as shell script parameter not global env variable

Reply:
I guess, although it means adding the possibility that you call them in the wrong order. Those variables appear with those names in lots of places already...